.sync/Files.yml

```diff
@@ -171,6 +173,7 @@ group:
       microsoft/mu_silicon_intel_tiano
       microsoft/mu_tiano_platforms
       microsoft/mu_tiano_plus
+      microsoft/secureboot_objects

```

Comment:
The CONTRIBUTING.md file there is intentionaly different than Project Mu at the moment. This one we can ignore.